### PR TITLE
Adding two openfoam installs:

### PR DESCRIPTION
**Pull request by AlexisEspinosaGayosso**
- new containerised module openfoam-org-container/10
  - new bare-metal installation of openfoam/v2206

### CODE REVIEW

**Comment by AlexisEspinosaGayosso** [comment]
The openfoam-org:10 image is already in quay, so the creation of the corresponding containerised module with shpc should work fine (as for other openfoam containers). Installation of the containerised module can proceed immediately.

The bare-metal install was ready since december but never went through as I did not updated the installation path to be an application. I have modified the modules.yaml file for that to be ready. Installation of the bare-metal vesion can proceed immediately.

**Review by dipietrantonio — APPROVED** [review]

**Comment by marcodelapierre** [comment]
@marcodelapierre

File: systems/setonix/settings.sh
Comment (marcodelapierre) [inline systems/setonix/settings.sh:136]: @AlexisEspinosaGayosso , for this work, you will need to submit a PR to https://github.com/singularityhub/shpc-registry, in order to add a YAML file for version 10, `10.yaml`, containing the list of executables, inside https://github.com/singularityhub/shpc-registry/tree/main/quay.io/pawsey/openfoam-org/aliases

**Review by marcodelapierre — CHANGES_REQUESTED** [review]
The changes are good, but the addition of the container for OF-org v10 requires a concurrent PR to the shpc-registry github repo

File: systems/setonix/settings.sh
Comment (AlexisEspinosaGayosso) [reply systems/setonix/settings.sh:135]: The list of aliases has been updated and the PR to the singularityhub has been requested.

**Comment by AlexisEspinosaGayosso** [comment]
PR to the singularityhub requested an update to use full paths in the aliases, this has been done and re-submitted

**Comment by AlexisEspinosaGayosso** [comment]
The `settings.sh` file has been updated to only install as modules those containers that have been rebuilt with mpich base image from ubuntu20.04.

**Comment by AlexisEspinosaGayosso** [comment]
PR to the singularityhub is now waiting for a review from @marcodelapierre

**Pull request merged**
